presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/test/RuleTester.java

```diff
@@ -33,11 +34,19 @@
 
     public RuleTester()
     {
-        session = testSessionBuilder()
+        this(ImmutableMap.of());
+    }
+
+    public RuleTester(Map<String, String> sessionProperties)
```

Comment:
setSystemProperty() has recently been added to RuleAssert, so this shouldn't be needed anymore